### PR TITLE
add full changelog using same format of cumulus core

### PR DESCRIPTION
**Pull request by sethvincent**
This creates a CHANGELOG.md file with information for all releases and uses the same format (https://keepachangelog.com/en/1.0.0/) as cumulus core.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Changelog
+All notable changes to this project will be documented in this file.
```
Comment (abarciauskas-bgse) [inline CHANGELOG.md]: I would change `will be documented` to `are documented`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```
Comment (abarciauskas-bgse) [inline CHANGELOG.md]: Minor suggestion to break this into 2 sentences.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+## [Unreleased]
+
+### Fixed
+- move CHANGES.txt to CHANGELOG.md with content for all releases, using same format as cumulus core
```
Comment (abarciauskas-bgse) [inline CHANGELOG.md]: Since this isn't a functional change I don't think it's necessary

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+### Added
+- docs: add troubleshooting section to readme
+
+### Fixed
+- fix `sys.exit` indentation
+
+## [v1.0.2] - 2018-02-28
+
+### Fixed
+- docs: fix typo
```
Comment (abarciauskas-bgse) [inline CHANGELOG.md]: Again - don't think we need to list non-functional changes

**Comment by abarciauskas-bgse** [comment]
Thanks Seth - I made a few more comments let me know what you think.

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+## [v0.0.7] - 2018-02-14
+
+### Fixed
+- update fix json schema validation to read from task root directory by default
+
+## [v0.0.6] - 2018-02-13
+
+### Added
+- add jsonschema validation
+- docs: descriptions of cumulus message schemas and the `createNextEvent` output
```
[the remark this answers is not in the document]
Comment (sethvincent) [reply CHANGELOG.md]: I think including things like typo fixes to docs is unnecessary, but usually find it useful for changelogs to include any kind of update to the project. I typically reserve the `Changed` and `Removed` sections to only include breaking changes to the actual code, and will include other kinds of updates in `Fixed` and `Added`, mirroring semver version bumps.

**Comment by sethvincent** [comment]
Updated to improve my sloppy indentation-related descriptions

**Pull request merged**